docs/admin/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- luxas
+- timothysc
+- fabriziopandini
+reviewers:
+- luxas
+- timothysc
+- fabriziopandini
+labels:
+- kind/documentation
```

Comment:
Can we also add the SIG label here?

Should it be `sig/cluster-lifecycle` or `sig/docs`?

Reply:
hm, if we add sig/cluster-lifecycle it would also apply the label to PRs that touch the `kubelet, kube-api-server, cloud-controller-manager, etc` files.

Reply:
ah, right :+1: